renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -1123,12 +1125,18 @@ void WrappedOpenGL::ActivateContext(GLWindowingData winData)
       // For now we assume we'll only get GL commands from a single thread
       QueuedInitialStateFetch fetch;
       fetch.res.Context = winData.ctx;
+      unsigned before = (unsigned)m_QueuedInitialFetches.size();
```

Comment:
I'd prefer `uint32_t` here. I don't mind `int` when the specific type doesn't really matter like a for loop counter, but generally I stick to the stdint types for any other cases (i.e. avoiding `long`, `unsigned int`, etc).

Reply:
I noticed there's already an instance of %zu being used to format size_t, so I switched these to it instead.  It's what I should have done originally, but I was being lazy about figuring out supported format strings.

Let me know if you'd still prefer uint32_t.

Reply:
No that's fine too, I didn't want to go down the rabbit's hole of "here's the custom printf specifiers I have" which honestly I forget about sometimes :smile:.